UnrealPlugin/TestingProject/Plugins/PlayFabGSDK/Source/PlayFabGSDK/Private/GSDKInfo.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
```

Comment:
I don't know the structure of these projects, but is there no way to share this header file instead of writing it twice?

Reply:
Same goes for the implementation (GSDKInternal.cpp) that is also duplicated

Reply:
Unfortunately not. When I first created the test project, I tried referencing the plugin in various ways, but you seem to have to copy the source to this path. Technically, we could delete the top level source, and this would be the only location, but it's just so deep in the project structure as to be annoying and maybe confusing to find, and it's always been available from the top level location too. But I'd be open for a case to be made to delete the top level source despite those points